test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -79,7 +79,7 @@ tar -czf "$local_code_archive_file" .
 gsutil cp "$local_code_archive_file" "$remote_code_archive_uri"
 
 # Install ksonnet
-KS_VERSION="0.11.0"
+KS_VERSION="0.13.1"
```

Comment:
I don't think this is a requirement for install kubeflow
https://www.kubeflow.org/docs/started/getting-started

Reply:
It's actually better that test uses minimum requirement to fail if earlier version is not working.

Reply:
What is the problem if I upgrade to use the latest version of ksonnet?